taipy/gui/gui.py

```diff
@@ -25,6 +24,7 @@
 import warnings
 from importlib import metadata, util
 from importlib.util import find_spec
+from inspect import currentframe, getabsfile, ismethod, ismodule, isroutine
```

Comment:
Yeah well... I don't really like those explicit imports because they hide the origin package, but hey... let's move on

Reply:
easier to read